app/abilities/contact.js

```diff
@@ -17,8 +17,10 @@ export default Ability.extend({
             return true;
           case 'consortium_admin':
             return (
-              this.get('currentUser.provider_id') ===
-              this.get('model.provider.consortium.id')
+              ((this.get('model.provider.memberType') === 'consortium_organization') &&
```

Comment:
This big conditional block is repeated 4 times.  Can we refactor this into a helper function so we can have the logic in one place if we need to understand/update it in the future.

Reply:
@jrhoads I have moved that code to a utility 'isAbleTo'.

Reply:
Looks good 👍